Packs/RecordedFuture/Integrations/RecordedFutureLists/RecordedFutureLists_test.py

```diff
@@ -150,6 +150,27 @@ def test_call_response_processing_return_error(self, mocker):
 
         mock_return_error.assert_called_once_with(message='mock error')
 
+    def test_call_raising_demisto_exception(self, mocker):
+        from CommonServerPython import DemistoException
+        import demistomock as demisto
+
+        mock_command_name = 'command_name'
+        mock_command_args = {'arg1': 'arg1_value', 'arg2': 'arg2_value'}
+
+        mocker.patch.object(demisto, 'command', return_value=mock_command_name)
+        mocker.patch.object(demisto, 'args', return_value=mock_command_args)
+        client = create_client()
+        mocked_http_request = mocker.patch.object(client, "_http_request")
+        mocked_http_request.side_effect = DemistoException(
+            "Some exception as a side effect"
+        )
+        raised = False
+        try:
```

Comment:
please use pytest.raises,  see Packs/AWS-ILM/Integrations/AWSILM/AWSILM_test.py (line 463) for example

Reply:
done